### PR TITLE
 Convert all Issue Templates to Issue Forms #81

### LINKED ISSUE #81: Convert all Issue Templates to Issue Forms
**Describe the solution you'd like**
An issue form with input and textarea fields, like the one that EddieHub has, [linked here.](https://github.com/EddieHubCommunity/BioDrop/issues/new/choose)

**Additional Context**
The fields would still be the ones we have in our Issue templates, only the formatting will have to match the aforementioned link.

- [ ] Would you like to work on this?

### PR DESCRIPTION
**Pull request by Latish705**
## Motivation and Context

Fixes #81 

## Description

I have created a new yml files for creating issue for bug or feature the ask the user in detail what issue are then facing while using or what feature they are expecting 

## How Has This Been Tested?
Yes I have tested

screenshot below
image

I have tested it with github preview 

### bug form screenshot below

image

### feature form screenshot below
image

## Types of Changes

- [ ] Bug fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Latish705** [comment]
added a little changes

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+- type: dropdown
+  id: browsers
+  attributes:
+    label: What browsers are you seeing the problem on?
+    multiple: true
+    options:
+      - Firefox
+      - Chrome
+      - Safari
+      - Microsoft Edge
```
Comment (agriyakhetarpal) [inline .github/ISSUE_TEMPLATE/bug_report.yml:54]:
```suggestion
      - Microsoft Edge

```

It's usually a good idea to [add newlines at the end of files](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: textarea
+  attributes:
+    label: Current Behavior
+    description: A concise description of what you're experiencing.
+  validations:
+    required: false
```
Comment (agriyakhetarpal) [inline .github/ISSUE_TEMPLATE/bug_report.yml:18]:
```suggestion
    required: true
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+name: 🐞 Bug
+description: File a bug/issue
+title: "[BUG] <title>"
```
Comment (agriyakhetarpal) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]:
```suggestion
title: "[BUG]: <title>"
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  attributes:
+    label: Steps To Reproduce
+    description: Steps to reproduce the behavior.
+    placeholder: |
+      1. In this environment...
+      2. With this config...
+      3. Run '...'
+      4. See error...
+  validations:
+    required: false
```
Comment (agriyakhetarpal) [inline .github/ISSUE_TEMPLATE/bug_report.yml:35]:
```suggestion
    required: true
```

File: .github/ISSUE_TEMPLATE/feature_report.yml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    id: extrainfo
+    attributes:
+      label: If "yes" to above, please explain how you would technically implement this
+      description: For example reference any existing code
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        You can also join our Discord community [here]
```
Comment (agriyakhetarpal) [inline .github/ISSUE_TEMPLATE/feature_report.yml:55]:
```suggestion
        You can also join our Discord community [here]

```

Same as above

File: .github/ISSUE_TEMPLATE/feature_report.yml [hunk 6]
```diff
@@ -0,0 +1,55 @@
+name: 💡 General Feature Request
+description: Have a new idea/feature for rbjks? Let us know...
+title: "[FEATURE] <description>"
```
Comment (agriyakhetarpal) [inline .github/ISSUE_TEMPLATE/feature_report.yml:3]:
```suggestion
title: "[FEATURE]: <description>"
```

File: .github/ISSUE_TEMPLATE/feature_report.yml [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+labels: ["⭐ goal: addition", "🚦 status: awaiting triage"]
```
Comment (agriyakhetarpal) [inline .github/ISSUE_TEMPLATE/feature_report.yml:4]: @adityaraute I need your opinion on the labels here, they look fine to me but up to you if you want emojis or a different label text

**Review by agriyakhetarpal — COMMENTED** [review]
Thanks for this @Latish705, there are a few comments that you can resolve. The rest looks good to me.

File: .github/ISSUE_TEMPLATE/feature_report.yml (hunk 7 above)
Comment (Latish705) [reply .github/ISSUE_TEMPLATE/feature_report.yml:4]: @adityaraute @agriyakhetarpal resolved all the changes wait for emoji response for pull request

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (Latish705) [reply .github/ISSUE_TEMPLATE/bug_report.yml:54]: done

Comment (agriyakhetarpal) [reply .github/ISSUE_TEMPLATE/bug_report.yml:54]: I don't think you have pushed the changes to your branch here

Comment (Latish705) [reply .github/ISSUE_TEMPLATE/bug_report.yml:54]: Should I raise a pull request

Comment (agriyakhetarpal) [reply .github/ISSUE_TEMPLATE/bug_report.yml:54]: No, you can keep contributing to the pull request (this current one) by pushing further changes to your branch

Comment (Latish705) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: add and thanks

File: .github/ISSUE_TEMPLATE/feature_report.yml (hunk 7 above)
Comment (adityaraute) [reply .github/ISSUE_TEMPLATE/feature_report.yml:4]: @Latish705 I don't think we need these labels here. Can you instead replace these with a simple label called `enhancement`

We can maybe think more on this later, but for now let's keep it simple.

Comment (Latish705) [reply .github/ISSUE_TEMPLATE/feature_report.yml]: @adityaraute done

**Comment by agriyakhetarpal** [comment]
Thanks @Latish705!

**Review by agriyakhetarpal — APPROVED** [review]
Thanks @Latish705!

**Comment by adityaraute** [comment]
Thanks for the contribution!

**Comment by Latish705** [comment]
@adityaraute @agriyakhetarpal my pleasure to contribute thanks for humble and good interaction

**Pull request merged**